src/networking.c

```diff
@@ -2589,6 +2589,13 @@ void pauseClients(mstime_t end) {
     server.clients_paused = 1;
 }
 
+/* Unpause clients. Used after a failover is complete to start processing
+ * commands again. */
+void unpauseClients() {
```

Comment:
there are a dew places that call `clientsArePaused` just for the "unpause side-effect"
now they can call `unpauseClients` directly